### DISCUSSION TITLE
Only noise coming through, dropping server

### DISCUSSION THREAD

**Discussion by carldms45** [jamulussoftware/jamulus#3266] [Technical Support & Help] [answered]
Hello,

One of my bandmates is having sound issues I can't diagnose and would appreciate some suggestions.

He is using Jamulus 3.10 on a Mac running Monterey with a Focusrite Scarlett 2i2.  Set up for audio in and out from the Scarlett.

I am running a private server on my Mac running Monterey.  One of my other bandmates can join in with no problem.

My friend can log on to my server but the only thing that comes through is a high-pitched buzz.  He can't hear anything from us.  The same thing happens to him if we use one of the public servers.  Last week he would only be logged in to my server for a minute or so and then he would be dropped.

This setup did work with a previous version of Jamulus.  We are in different parts of the country but the delays don't seem that bad (50 ms?).

We tried just using the "built in" audio option but the same thing happened.

He had to manually confirm the integrity of Jamulus when it was opened (but so did I).

He is not running a "hardwire" for ethernet (so on wi-fi) but that wasn't a big problem previously.  I am on a hardwire.

One suggestion I saw is that he could try running his own server and logging in to that (can I do that without messing with any of the modem ports?)

Restarting the computer to update some of the security settings?  (I did see the message in Troubleshooting about enabling microphone through Security when using a Mac.  Is there something similar for the Scarlett?)

Thanks,

Carl

**Comment by chrisrimple** [comment]
First thing I'd check is the bitrate.  From Applications > Utilities, run the Audio MIDI Setup app.  From the Window menu, choose Show Audio Devices (if that window is not already visible).  Click on each audio device in the list on the left and ensure that both Input and Output are set to 48k.  Jamulus requires 48k, so if the Focusrite isn't set to 48k, it won't work.

**Reply by carldms45** [reply]
Thanks for the suggestion.  Those were set to 48.

**Reply by chrisrimple** [reply, marked as answer]
Since he's just getting noise regardless of which server is used, it has to be a problem on his end.  If he plugs in wired earbuds with microphone to his Mac, sets Apple > Preferences > Sound > Input and > Output to the wired earbuds, and sets Jamulus IN/OUT to System, he's also getting noise?  But other sound (outside Jamulus) to the wired earbuds is OK?  If true, then maybe try uninstalling/reinstalling Jamulus.  You might also want to try the "legacy" Jamulus build that you can [find at the bottom of the page here](https://github.com/jamulussoftware/jamulus/releases/tag/r3_10_0).

**Reply by carldms45** [reply]
Problem solved, thanks.  He believes it may have been a hardware issue since he sprayed some contact cleaner on the Scarlett and it started working.  He was getting LED indicators on the Scarlett previously but I guess the analog-digital converter wasn't getting sufficient signal.  I had previously tried the IN/OUT on the system; don't know why that didn't work.

Sorry for the extra work.  But no problems wth the software!

-Carl
